keyboards/kbdfans/kbd8x_mk2/keymaps/default_tkl_ansi/keymap.c

```diff
@@ -0,0 +1,43 @@
+/* Copyright 2019 Ryota Goto
```

Comment:
These should just be named `default_ansi`/`default_iso` since it overlaps with the layout keymap name.

Reply:
We should maybe talk about the [keyboard guidelines](https://docs.qmk.fm/#/hardware_keyboard_guidelines?id=ltkeyboard_namehgt) because I named these after how I understand the guidelines.